### PR TITLE
Make org name for endpoint security configurable

### PR DESCRIPTION
**Pull request by omnivion**
### Information
- JIRA story QPPCT-612.

### Changes proposed in this PR:
- Adds the ORG_NAME variable

### Checklist 
- [X] All JUnit tests pass (`mvn clean verify`).
- [X] New unit tests written to cover new functionality.
- [X] Added and updated JavaDocs for non-test classes and methods.
- [X] No local design debt. Do you feel that something is "ugly" after your changes?
- [X] Updated documentation (`README.md`, etc.) depending if the changes require it.

### CODE REVIEW

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java [hunk 1]
```diff
@@ -94,6 +107,6 @@ private String removeSignature(String jws) {
 	 */
 	private boolean isValidCpcPlusOrg(Map<String, String> payloadMap) {
 		String orgName = payloadMap.get("name");
```
Comment (FlexionSonarQube) [inline rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java]: MAJOR Rename "orgName" which hides the field declared at line 25. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)

**Comment by FlexionSonarQube** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java (hunk 1 above)
Comment (clydet) [reply rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java]: Not keen on this check ^

Comment (omnivion) [reply rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java]: Same @clydet.

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/config/SecurityConfig.java [hunk 2]
```diff
@@ -16,6 +18,9 @@
 
 	private static final String CPC_WILDCARD = "/cpc/**";
 
+	@Value("${ORG_NAME:" + JwtAuthorizationFilter.DEFAULT_ORG_NAME + "}")
```
Comment (clydet) [inline rest-api/src/main/java/gov/cms/qpp/conversion/api/config/SecurityConfig.java]: why not `@Value("${ORG_NAME}")`?

Comment (omnivion) [reply rest-api/src/main/java/gov/cms/qpp/conversion/api/config/SecurityConfig.java]: @clydet you can give a default value with a :

File: rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java (hunk 1 above)
Comment (halprin) [reply rest-api/src/main/java/gov/cms/qpp/conversion/api/security/JwtAuthorizationFilter.java]: Watch out @FlexionSonarQube, you've got a mutiny of your hands...  :laughing:

**Pull request merged**